src/video.cpp

```diff
@@ -1281,6 +1281,17 @@ void encode_run(
   const encoder_t &encoder,
   void *channel_data) {
 
+  // get minimum fps from config
+  auto MIN_FPS = std::chrono::milliseconds(*&config::video.min_fps * 10);
+
+  // ensure MIN_FPS is at least 10 and less than 2400 milliseconds
+  if(MIN_FPS < std::chrono::milliseconds(10)) {
+    MIN_FPS = std::chrono::milliseconds(10);
+  }
+  else if(MIN_FPS > std::chrono::milliseconds(2400)) {
```

Comment:
We can just get rid of this and use an || operator instead following the suggestion I mentioned earlier.

Reply:
We don't need to do any validation here as it's taken care of in the `apply_config` function of `config.cpp`.